modules/custom/openy_activity_finder/templates/openy-activity-finder-program-search.html.twig

```diff
@@ -7,119 +7,35 @@
  */
  *
 #}
+
 {{ attach_library('openy_activity_finder/openy_activity_finder') }}
 
 <div id="activity-finder-app">
-  <div class="container activity-finder__wrapper">
+  <div class="activity-finder__wrapper">
 
-    <div v-show="step === 0" class="activity-finder__step_0 activity-finder__step">
-      <h1>{{ 'Program Search'|t }}</h1>
-      <form role="search">
-        <div>
-          <div class="form-item form-item-textfield">
-            <label for="edit-name-search-programs">{{ 'How would you like to search for programs?'|t }}</label>
-            <input type="text" class="form-text" v-model="keywords" id="edit-name-search-programs" placeholder="{{ 'Search for programs on your terms ...'|t }}" />
-          </div>
-          <div class="form-actions">
-            <input @click.prevent="next()" type="submit" class="form-submit" />
-          </div>
-        </div>
-      </form>
-      <div class="actions">
-        <button @click.prevent="toStep(1)" class="btn btn-primary btn-lg">{{ 'Activity Finder'|t }}<span>{{ 'Find exactly what you-re looking for'|t }}</span></button>
-        <a v-bind:href="afResultsRef" class="btn btn-primary btn-lg">{{ 'View all programs'|t }}<span v-if="loading">Loading...</span><span v-else>${ table.count } {{ 'results'|t }}</span></a>
-      </div>
+    {# Step 1 #}
+    <div v-show="step === 0" class="activity-finder__step">
+      {% include '@openy_activity_finder/steps/openy-activity-finder-step-1.html.twig' %}
```

Comment:
@aaronjchristian Please let's not split html involved in vue application into separate twig templates. This will make maintenance of the vue app very hard. If we like to split it into components we need to use vue components rather than separate twig files.

Reply:
Waiting for further discussion around this one.